### PR TITLE
Milestone number 2: request test

### PR DESCRIPTION
**Pull request by eroiyo**
Hello Microverse community, this is the milestone number 2, the next thing were implemented

- [ ] test to see if the posts request is successfully

- [ ] test to see if the usersrequest is successfully

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Posts', type: :request do
+  it 'return true if display' do
+    get '/users/1/posts'
+
+    expect((response)).to have_http_status(:success)
```
Comment (Davidosky007) [inline spec/requests/posts_spec.rb:9]: * Kindly add the necessary tests as stated in the project requirement currently your test lacks an index and show actions tests, ensures correct view template is rendered and  response body includes the correct placeholder text
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @eroiyo 👋, 

Good job so far! 💪 
* Well written PR ✔️ 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @eroiyo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Thanks!, I will

**Pull request merged**